shared/structures/src/Platform.ts

```diff
@@ -532,6 +533,23 @@ export class Platform extends AutoEncoder {
         return this.privateConfig?.roles ?? [];
     }
 
+    /**
+     * Throws an error if userMode is not platform.
+     * The period id of the platform should almost never be used if the userMode is not platform.
+     * By throwing an error, we prevent accidental usage of the period id.
+     */
+    get periodIdIfPlatform() {
```

Comment:
Voor zover ik kan zien wordt dit niet gebruikt in de frontend?

Ik weet niet of we dit echt zo moeten doen. Lijkt me echt gevaarlijk om op random plaatsen een error te gooien in de frontend, dan stopt die gewoon met werken.